CHANGELOG.md

```diff
@@ -30,6 +30,8 @@ Release channels have their own copy of this changelog:
 * `cargo-build-sbf` and `cargo-test-sbf` now accept `v0`, `v1`, `v2` and `v3` for the `--arch` argument. These parameters specify the SBPF version to build for.
 * SBFPv1 and SBPFv2 are also available for Anza's C compiler toolchain.
 * SBPFv3 will be only available for the Rust toolchain. The C toolchain will no longer be supported for SBPFv3 onwards.
+* Although the solana rust toolchain still supports the `sbf-solana-solana` target, the new `cargo-build-sbf` version target defaults to `sbpf-solana-solana`. The generated programs will be available on `target/deploy` and `target/sbpf-solana-solana/release`.
```

Comment:
Can you add information or a sample command about how to use the `sbf-solana-solana` target?

Also, can you put this line under a new `#### Breaking` heading, similar to how it's done in other releases?

Reply:
Done in https://github.com/anza-xyz/agave/pull/5227/commits/7176f3ad1de4901a7c84077f0a9951d5b7cc035a

Reply:
I think you still have one too many `#` in there; should be 4 of them

Reply:
Removed one hashtag.